### PR TITLE
Update design to BP details page

### PR DESCRIPTION
**Pull request by AngeloCG97**
### Update design to BP details page 

#657

### Steps to test
1. Update bp profile design: Desktop and Mobile

### CODE REVIEW

**Review by xavier506 — COMMENTED** [review]
Where will we get eden data ? will this work if there are no eden ratings?

**Comment by AngeloCG97** [comment]
At the moment this data is not being consulted, the objective of this PR is to improve the design of this section and take advantage to include the new components for Eden, I will create a new issue to build the necessary backend to extract the information from Eden.

File: webapp/src/language/es.json [hunk 1]
```diff
@@ -180,6 +180,9 @@
 
   "profile": {
     "generalInformation": "Información General",
+    "eosRates": "Clasificación EOS",
```
Comment (leisterfrancisco) [inline webapp/src/language/es.json:183]: Clasificación o Calificación? Is that right?

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

File: webapp/src/language/es.json (hunk 1 above)
Comment (xavier506) [reply webapp/src/language/es.json]: Calificación

**Pull request merged**